lib/chefdepartie/cookbook.rb

```diff
@@ -28,6 +33,7 @@ def self.upload_cookbooks(path, books)
 
       books = books.collect do |name|
         next if Cache.cache(File.join(path, name))
+        next if File.file?(File.join(path, name))
```

Comment:
http://ruby-doc.org/core-2.2.0/File.html#method-c-file-3F

> Returns true if the named file exists and is a regular file.
>
> file can be an IO object.
>
> If the file argument is a symbolic link, it will resolve the symbolic link and use the file referenced by the link.